src/Web/Intercom/Types.hs

```diff
@@ -50,23 +52,40 @@ ping c = do
 opts c = defaults & auth ?~ basicAuth (c ^. appId) (c ^. apiKey)
                   & header "Accept" .~ ["application/json"]
 
+postOpts c = opts c & header "Content-Type" .~ ["application/json"]
+
 data User = User {
   _name             :: Maybe Text,
   _email            :: Maybe Text,
   _userId           :: Maybe Text,
-  _customAttributes :: Map String Value
+  _customAttributes :: HashMap Text Value
 } deriving (Eq, Show)
 
+-- | A blank user, for overriding
+-- > let myUser = blankUser & email .~ (Just "bob@foo.com")
+blankUser :: User
+blankUser = User Nothing Nothing Nothing (fromList [])
```

Comment:
this is just an empty User users can override with their details as the doc comment shows